apps/avifenc.c

```diff
@@ -1945,7 +1945,7 @@ int main(int argc, char * argv[])
         }
     }
 
-    avifIOStats ioStats = { 0, 0, 0 };
+    avifIOStats ioStats = { 0, 0 };
```

Comment:
Maryla: It would be good to modify the following code in avifenc.c:

```
    printf("Encoded successfully.\n");
    printf(" * Color AV1 total size: %" AVIF_FMT_ZU " bytes\n", ioStats.colorOBUSize);
    printf(" * Alpha AV1 total size: %" AVIF_FMT_ZU " bytes\n", ioStats.alphaOBUSize);
```

to also print the gain map AV1 total size, and see if you find it useful to report that information.

Reply:
Actually, there's no way to encode a gain map with avifenc right now, only through the API, so this can be done later I think.